layout-management-services/src/main/java/org/exoplatform/layoutmanagement/rest/SiteManagementRestService.java

```diff
@@ -128,13 +132,20 @@ public Response updateSite(@Parameter(description = "site type")
                              String siteLabel,
                              @Parameter(description = "site description")
                              @QueryParam("siteDescription")
-                             String siteDescription) {
+                             String siteDescription,
+                             @Parameter(description = "site displayed in meta site")
+                             @QueryParam("displayed")
+                             boolean displayed,
+                             @Parameter(description = "site display order")
+                             @QueryParam("displayOrder")
+                             int displayOrder) {
     try {
-      SiteId siteId = Util.from(new SiteKey(siteType, siteName));
-      Site site = portal.getSite(siteId);
-      site.setDescription(siteDescription);
-      site.setDisplayName(siteLabel);
-      portal.saveSite(site);
+      PortalConfig portalConfig = layoutService.getPortalConfig(new SiteKey(siteType, siteName));
+      portalConfig.setDescription(siteDescription);
+      portalConfig.setLabel(siteLabel);
+      portalConfig.setDisplayed(displayed);
+      portalConfig.setDisplayOrder(displayed ? displayOrder : 0);
+      layoutService.save(portalConfig);
```

Comment:
IMO it is better to work directly with Site to be updated with the new attributes